### PR TITLE
feat: Add examples for email template profile

### PR DESCRIPTION
**Pull request by smizell**
## Description

Add examples for `communication/email-template`. I copied them from the tests. My only concern is with the long HTML and text examples. Should we do this or make them short one-liners?

## Motivation and Context

Trying to add examples for all profiles.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by kysely** [comment]
@smizell Please don't merge it yet, I'll push a fix to the front end first. I'll ping you here

File: capabilities/communication/email-templates/profile.supr
Comment (kysely) [inline capabilities/communication/email-templates/profile.supr:150]:
```suggestion
      html = """<!doctype html>
<html>

<head>
  <meta name="viewport" content="width=device-width">
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <title>Your order has shipped!</title>
</head>

<body class="">
Hi there, we shipped your recent order on Our Shop. <a>You can track it here</a>.
</body>

</html>
"""
```

File: capabilities/communication/email-templates/profile.supr
Comment (kysely) [inline capabilities/communication/email-templates/profile.supr:135]:
```suggestion
      name = "Shipped Order Template"
      subject = "Your order has shipped!"
      text = "Hi there, your recent order on Our Shop has been shipped."
```

File: capabilities/communication/email-templates/profile.supr
Comment (kysely) [inline capabilities/communication/email-templates/profile.supr:155]:
```suggestion
      name = "Shipped Order Template"
```

File: capabilities/communication/email-templates/profile.supr
Comment (kysely) [inline capabilities/communication/email-templates/profile.supr:111]:
```suggestion
      name = "Shipped Order Template"
      subject = "Your order has shipped!"
      text = "Hi there, your recent order on Our Shop has been shipped."
      html = """<!doctype html>
<html>

<head>
  <meta name="viewport" content="width=device-width">
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <title>Your order has shipped!</title>
</head>

<body class="">
Hi there, we shipped your recent order on Our Shop. <a>You can track it here</a>.
</body>

</html>
"""
    }

    result {
      id = "d-8be96c63de164e6799c5dbf2521d7959"
      name = "Shipped Order Template"
```

**Review by kysely — APPROVED** [review]
I think this is good. The multi-line examples are stringified in the JSON examples anyways.

Screenshot 2021-11-11 at 19 29 09

One thing I'd suggest is to use a different _story_ behind the examples themselves. E.g. I used eCommerce use case of confirming order fulfilment in [`send-email`](https://superface.ai/communication/send-email) examples. I added the suggestions if you're interested in including them here, too.

**Comment by kysely** [comment]
> @smizell Please don't merge it yet, I'll push a fix to the front end first. I'll ping you here

I pushed the fix to production, this can be merged when it's ready.

File: capabilities/communication/email-templates/profile.supr
Comment (freaz) [inline capabilities/communication/email-templates/profile.supr:53]: @kysely I checked how it is rendered (and if :) ) in latest Air. And for this, highlighting seems to be a bit broken.

image

But example json seems to be valid. Do you think it is ok to merge?

Comment (kysely) [reply capabilities/communication/email-templates/profile.supr:53]: I know 😢 The highlighter is very naive (but lightweight) and there's an open discussion of whether we'll keep it or use a more robust one.

As for this case, I'd still merge the example now. Having it there feels more important than broken highlighting.

Comment (freaz) [reply capabilities/communication/email-templates/profile.supr:53]: 👍 I still think your suggestions should be worked in, so I will do it and maybe it will work better with highlighter

**Pull request merged**
